pkgs/development/tools/blink/default.nix

```
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub
+}:
+
+
+stdenv.mkDerivation (rec {
+  pname = "blink";
+  version = "0.9.2";
+
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "blink";
+    rev = "${version}";
+    hash = "sha256-GQ1uOZr75CXLQ6aUX4oohJhwsYy41StaZiQRNrD/FXQ=";
+  };
```

```suggestion
  };
  
  configureFlags = [
    "CC=${stdenv.cc.targetPrefix}cc"
  ];
```

You are sure we do not want ``makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];`` instead?